### DISCUSSION TITLE
How to remove MonitorControl

### DISCUSSION THREAD

**Discussion by bestyys** [MonitorControl/MonitorControl#1333] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Anyone can help that, I just want to know how to remove MonitorControl from my Mac, as I cannot remove it via common way...

**Comment by waydabber** [comment]
You need to move the app from Applications to the Trash using the Finder and then empty the Trash.

**Reply by pinacate** [reply]
I would also like to remove MonitorControl and any associated support files from my Mac. 

Dragging or any of the other standard ways to remove MonitorControl.app don't work as the file is always 'In Use'. What's the proper way to uninstall the app and associated components?

Thanks!

**Reply by pinacate** [reply]
Never mind. The preference was set to launch at startup. I changed that setting, rebooted and then was able to delete the app. 
Thanks!
